tests/test_phase_psf.py

```diff
@@ -1265,6 +1270,90 @@ def test_shared_memory():
             initargs=galsim.phase_screens.initWorkerArgs()
         )
 
+@timer
+def test_pickle():
+    # This is response to ImSim issue #234
+    # https://github.com/LSSTDESC/imSim/issues/234
+
+    import multiprocessing
+
+    # Pull out the relevant bits from the imsim code base.
+
+    rng = galsim.BaseDeviate(1234)
+    ud = galsim.UniformDeviate(rng)
+    gd = galsim.GaussianDeviate(rng)
+
+    altitudes = [0.0, 2.58, 5.16, 7.73, 12.89, 15.46]
+    altitudes[0] = 0.2
+
+    weights = [0.652, 0.172, 0.055, 0.025, 0.074, 0.022]
+    weights = [np.abs(w*(1.0 + 0.1*gd())) for w in weights]
+    weights = np.clip(weights, 0.01, 0.8)  # keep weights from straying too far.
+    weights /= np.sum(weights)  # renormalize
+
+    L0 = 0
+    while L0 < 10.0 or L0 > 100:
+        L0 = np.exp(gd() * 0.6 + np.log(25.0))
+    r0_500 = 0.2
+
+    L0 = [L0]*6
+
+    maxSpeed = 20.0
+    speeds = [ud()*maxSpeed for _ in range(6)]
+    directions = [ud()*360.0*galsim.degrees for _ in range(6)]
+
+    #screen_size = 819.2
+    screen_size = 32
+    screen_scale = 0.1
+    kmax = 1
+
+    if sys.version_info >= (3,4):
+        import multiprocessing as mp
+        if __name__ == "__main__":
+            ctxs = [None, mp.get_context("fork"), mp.get_context("spawn"), "forkserver"]
```

Comment:
All four only take 0.4 seconds to run this test on my laptop, so probably don't need to bother with this distinction.  Can just always do all of them in py3.